content/en/docs/aws/rds.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Amazon RDS"
```

Comment:
Em. `Amazon RDS` will be one of the items in the list. I feel it doesn't match with existing titles. What about something like managed database, etc.

Reply:
I changed it to 'Configre External Database Using Amazon RDS'. WDYT?

Reply:
Sounds good to me